src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj

```diff
@@ -25,7 +25,65 @@
     <Folder Include="Properties\" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Update="ProjectSystem\VS\PropertyPages\DebugPageControl.xaml.cs">
+    <Compile Include="GlobalSuppressions.cs" />
```

Comment:
This isn't right. We shouldn't be adding all these files - the default globbing will pick them up. We just need <Compile Update="ProjectSystem\VS\PropertyPages\DebugPageControl.xaml.cs"> to add the DependentUpon attribute,